client/src/App.js

```diff
@@ -18,22 +19,20 @@ const App = () => (
   <>
     <GlobalStyles />
     <FetchUser>
-      <Navbar />
-      <Container as={AppContainer}>
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route exact path="/login" component={Login} />
-          <Route exact path="/register" component={Register} />
-          <ProtectedRoute
-            exact
-            path="/units/:unit_id/contents/:id"
-            component={ContentView}
-          />
-          <ProtectedRoute exact path="/dashboard" component={Dashboard} />
-          <ProtectedRoute exact path="/courses/:id" component={CourseWork} />
-          <Route component={NoMatch} />
-        </Switch>
-      </Container>
+      <Navbar>
+        <Container as={AppContainer}>
+          <Switch>
+            <Route exact path='/' component={Home} />
+            <Route exact path='/login' component={Login} />
+            <Route exact path='/register' component={Register} />
+            <ProtectedRoute exact path='/units/:unit_id/contents/:id' component={ContentView} />
+            <ProtectedRoute exact path='/dashboard' component={Dashboard} />
+            <ProtectedRoute exact path='/profile' component={Profile} />
+            <ProtectedRoute exact path='/courses/:id' component={CourseWork} />
+            <Route component={NoMatch} />
+          </Switch>
+        </Container>
+      </Navbar>
```

Comment:
We don't want the closing Navbar.I think this was fixed but maybe change that to be safe.

Reply:
Fixed the issue